libs/market-depth/src/lib/orderbook-row.tsx

```diff
@@ -43,19 +43,19 @@ export const OrderbookRow = React.memo(
         <Vol
           testId={`bid-vol-${price}`}
           value={bid}
-          valueFormatted={addDecimal(bid, positionDecimalPlaces)}
+          valueFormatted={addDecimal(bid, decimalPlaces)}
```

Comment:
question (blocking): Are you sure these three changes are correct? My understanding is that price values should use market.decimalPlaces, and volume/size should use market.positionDecimalPlaces (for fractional order sizes)

Reply:
That's on me. I had it wrong in the ticket. have corrected it now.

Reply:
changed this to align with the criteria